app/uk/gov/hmrc/gform/sharedmodel/email/LocalisedEmailTemplateId.scala

```diff
@@ -45,4 +48,6 @@ object LocalisedEmailTemplateId {
     case otherwise =>
       JsError("Invalid email template id definition. Expected json String or json Object, but got: " + otherwise)
   }
+
+  implicit val format: OFormat[LocalisedEmailTemplateId] = OFormatWithTemplateReadFallback(reads)
```

Comment:
Cannot we replace existing `reads` and new `format` with just:

```
implicit val format: Format[LocalisedEmailTemplateId] = derived.oformat()
```

There is no reason to use `OFormatWithTemplateReadFallback` on frontend.